### PR TITLE
Uiregistration

### PR DESCRIPTION
**Pull request by Chanpech**
Just got done finalizing the last few things on the registration page. The <link href=> for "Don't have an account? Sign Up" should take you to the registration page and vice versa for "Already have an account". We can remove the register from the hamburger menu if you deem it necessary. There shouldn't be any more of the prettier/prettier compile error.

### CODE REVIEW

**Review by tjzetty — APPROVED** [review]
Good implementation of the register page. Should be super easy to hook up.

File: .prettierignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 # build
 dist/
-build/
\ No newline at end of file
+build/
+␍/
```
Comment (tjquillan) [inline .prettierignore]: Lets not ignore / for formatting. Might ignore everything? Not quite sure...

File: apps/frontend/src/components/App.tsx [hunk 2]
```diff
@@ -69,6 +70,7 @@ export default function App() {
         >
           <List>
             <ListItemNavLink to="/" primary="Home" />
+            <ListItemNavLink to="/register" primary="Register" />
```
Comment (tjquillan) [inline apps/frontend/src/components/App.tsx]: Register should only be sent to by buttons / direct links. Not the navbar

File: apps/frontend/src/components/App.tsx [hunk 3]
```diff
@@ -84,7 +86,9 @@ export default function App() {
       >
         <Routes>
           <Route path="/login" element={<Login />} />
+          <Route path="/register" element={<Register />} />
           <Route path="/" element={<Home />} />
+          <Route path="/register" element={<Register />} />
```
Comment (tjquillan) [inline apps/frontend/src/components/App.tsx]: Duplicate routs?

File: apps/frontend/src/pages/Login.tsx [hunk 4]
```diff
@@ -73,7 +73,7 @@ export default function Login() {
               </Link>
             </Grid>
             <Grid item>
-              <Link href="#" variant="body2">
+              <Link href="/register" variant="body2">
```
Comment (tjquillan) [inline apps/frontend/src/pages/Login.tsx]: This doesn't seem to use react-roughter

File: apps/frontend/src/pages/Register.tsx [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+            type="submit"
+            fullWidth
+            variant="contained"
+            sx={{ mt: 3, mb: 2 }}
+          >
+            Register
+          </Button>
+          <Grid container>
+            <Grid item>
+              <Link href="/login" variant="body2">
```
Comment (tjquillan) [inline apps/frontend/src/pages/Register.tsx]: Should also use react-router

**Review by tjquillan — CHANGES_REQUESTED** [review]

**Review by tjquillan — APPROVED** [review]

**Pull request merged**